inst/help/selectionmodels.md

```diff
@@ -1,18 +1,18 @@
-Robust Bayesian Meta-Analysis
+Selection Models
 ===
 
-The selection models allow the user to specify a publication bias adjusted random and fixed effects meta-analytic models. The options allow the user to specify the selection process using either one or two-sided p-value cutoffs, which are internally transformed to one-sided p-value cutoffs prior to the fitting process.
+The selection models are meta-analytic methods on the fixed and random effects meta-analytic models that adjust for publication bias operating on p-values (Iyengar & Greenhouse, 1988; Vevea & Hedges, 1995). The analysis allow users to specify the selection process using either one or two-sided p-value cutoffs, which are internally transformed to one-sided p-value cutoffs prior to the fitting process.
```

Comment:
Maybe remove `The` at the start of the sentence? Maybe I am reading this wrong though

Reply:
No, you are reading it correctly. It's just my bad English.